src/Request.php

```diff
@@ -494,4 +499,20 @@ private function getFormatter()
     {
         return $this->client->getFormatter($this->getFormat());
     }
+
+    /**
+     * @return resource
+     */
+    public function getFile()
```

Comment:
Do we need a getter?

Reply:
Why not? But I can remove it, if you recommend so.

Reply:
It's better not to add it if we don't have any use case for it.

Reply:
Actually, we use it in CurlTransport class.

Reply:
Right. Never mind then.